lib/evss/disability_compensation_form/data_translation_all_claim.rb

```diff
@@ -170,32 +170,17 @@ def list_attachment_filenames(attachments)
       ###
       # Banking info
       ###
-
       def translate_banking_info
         monitor = DisabilityCompensation::Loggers::Monitor.new
-
         populated = input_form['bankName'].present? && input_form['bankAccountType'].present? &&
                     input_form['bankAccountNumber'].present? && input_form['bankRoutingNumber'].present?
-        # If banking data is not included, then it has not changed and will be retrieved from Lighthouse
         if !populated || redacted(input_form['bankAccountNumber'], input_form['bankRoutingNumber'])
-
-          # NOTE: if the Veteran supplied empty or redacted banking information, we are removing a call to Lighthouse
-          # to retrieve banking info the Veteran has on file with Lighthouse, to respect the fact the Veteran left
-          # this blank on purpose.
-          #
-          # This change will launch behind a Flipper but the Flipper will eventually be removed and we will return an
-          # empty hash here
-          if Flipper.enabled?(:disability_526_block_banking_info_retrieval)
-            monitor.track_526_submission_without_banking_info(@user.uuid)
-            {}
-          else
-            get_banking_info
-          end
+          # Respect the fact the Veteran left banking information blank on purpose
+          # by not retrieving their existing banking info from Lighthouse
```

Comment:
Nit: we can remove this note because we are removing the Lighthouse call so this is kind of moot and might be confusing for someone who wasn't aware of the prior behavior